### PR TITLE
OH2-363 | Remove permissions CRUD Core methods

### PR DESCRIPTION
**Pull request by SilverD3**
Removal of unused methods related to [OH2-363](https://openhospital.atlassian.net/browse/OH2-363)

[OH2-363]: https://openhospital.atlassian.net/browse/OH2-363?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/main/java/org/isf/permissions/manager/GroupPermissionManager.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 @Component
 public class GroupPermissionManager {
 
-	private GroupPermissionIoOperations operations;
+	private final GroupPermissionIoOperations operations;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/permissions/manager/GroupPermissionManager.java:36]: I do not see the value of adding `final`.  The variable is already `private` and adding `final` doesn't add anything to how the variable can be used.   In addition, if it is added, then there are hundreds of other occurrences like this in the code base that should be changed to match this pattern.   It isn't worth it IMHO.

Comment (SilverD3) [reply src/main/java/org/isf/permissions/manager/GroupPermissionManager.java:36]: Variables which values are set via DI (Dependency Injection) should be final as there are initialized once.

Comment (mwithi) [reply src/main/java/org/isf/permissions/manager/GroupPermissionManager.java:36]: Thank you @SilverD3, could you please open a "task" for this so we don't forget?

Comment (SilverD3) [reply src/main/java/org/isf/permissions/manager/GroupPermissionManager.java:36]: Sure. Issue OP-1325

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**